src/setup/validateTxnFields.ts

```diff
@@ -186,9 +186,12 @@ export const validateTxnFields =
       try {
         const transaction = getTransactionObj(tx)
         const isSigned = transaction.isSigned()
-        const { address: senderAddress, isValid } = getTxSenderAddress(transaction,txId) // ensures that tx is valid
-        const isSignatureValid = isValid
-        if (ShardeumFlags.VerboseLogs) console.log('validate evm tx', isSigned, isSignatureValid)
+        const {
+          address: senderAddress,
+          isValid: isAddressValid,
```

Comment:
this could be the signature or the address that are invalid. The name doesn't match the counter or the fail reason
`nestedCountersInstance.countEvent('shardeum', 'validate - sign failed')`
`reason: 'Transaction signature is invalid',`